service/e2e_tests/test_search.py

```diff
@@ -92,7 +92,7 @@ def test_search_did_you_mean(self):
             EC.presence_of_element_located((By.CSS_SELECTOR, "section.spinner")))
 
         WebDriverWait(self.selenium, self.explicit_wait).until(
-            EC.text_to_be_present_in_element((By.CSS_SELECTOR, ".not_found"), "Geen resultaten voor"))
+            EC.text_to_be_present_in_element((By.CSS_SELECTOR, ".not_found"), "Because-no-results-for"))
```

Comment:
Deze zou je eventueel kunnen toevoegen aan de locales fixture? Dan checkt het systeem ook of de vertalingen goed verwerkt worden. Letterlijk deze string zou niet goed zijn natuurlijk.